### PR TITLE
DOC: Continuous restructuring our docs

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
Making more clear the development strategy.

Waiting for #589 before this can be put in a mergeable state

### CODE REVIEW

**Comment by jovan-stojanovic** [comment]
#589 rather than #509? :)

File: doc/vision.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+target to predict to guide assembly when possible (supervised learing for
+data assembly).
+
+In the long term, as skrub is built on higher-level APIs, it will make it
+easier for data-scientist to use efficient database patterns and
+backends.
+
+Skrub seeks tradeoffs in terms of flexibility: its high-level APIs are by
+construction restrictive compared to directly manipulating dataframes.
+This is by design, as skrub does not aim to replace tools such as `pandas
```
Comment (Vincent-Maladiere) [inline doc/vision.rst:29]:
```suggestion
This is by design, as skrub does not aim to replace tools such as `Pandas
```

File: doc/vision.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+<https://pandas.pydata.org>`__, `Ibis <https://ibis-project.org>`__,
+`duckdb <https://duckdb.org/>`_.
```
Comment (Vincent-Maladiere) [inline doc/vision.rst:31]:
```suggestion
`DuckDB <https://duckdb.org/>`_.
```

**Review by Vincent-Maladiere — COMMENTED** [review]
I like the spirit of the roadmap very much! I noted a few typos ;)

File: doc/vision.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- Add a Join-aggregator object, to do feature augmentation on one-to-many
+  correspondances
+
+- fuzzy joining on datetime values
+
+- Support polars?
+
+- Support time series (eg in the aggregations)
+
+- Interpolator join to be able
```
[the remark this answers is not in the document]
Comment (GaelVaroquaux) [reply doc/vision.rst]: Fixed, thanks

**Comment by GaelVaroquaux** [comment]
OK, this should be good for merge.

I've done a quick fix for the lower part of the landing page. It's clearly to be improved, but I feel that it is better than what we currently have (and one challenge is always to have something that works both in light and dark theme)

**Review by Vincent-Maladiere — APPROVED** [review]
Looks good :)

**Comment by GaelVaroquaux** [comment]
Merging (I fixed a few links that were broken in master)

**Pull request merged**